src/main/java/com/dih/connector/test/client/connector/DataspaceConnectorConfiguration.java

```diff
@@ -15,4 +36,44 @@
     public BasicAuthRequestInterceptor adminAuth() {
         return new BasicAuthRequestInterceptor(username, password);
     }
+
+    @Bean
+    public BasicAuthenticationInterceptor authInterceptor() {
```

Comment:
Why do we need one more basic auth interceptor?

Reply:
BasicAuthenticationInterceptor != BasicAuthRequestInterceptor